src/modules/emotes/channel-emotes.js

```diff
@@ -22,10 +23,10 @@ class ChannelEmotes extends AbstractEmotes {
         return provider;
     }
 
-    updateChannelEmotes({urlTemplate, emotes}) {
+    updateChannelEmotes({channelEmotes}) {
```

Comment:
you are forgetting `sharedEmotes`

Reply:
Fixed, i didn't notice the separation in the new api. I wasn't sure if there was a more concise way to write the shared/channel emote loops, so there is a similar loop for each.